### DISCUSSION TITLE
Remember welcome window's position

### DISCUSSION THREAD

**Discussion by adamwilt** [videovillage/community#124] [Ideas]
I use the welcome window as a drop target for opening files. I'm currently downloading about 10,000 dpx files, 100 at a time, and checking them visually as I go. To do this, I drop each new unzipped folder of 100 files onto the welcome window, but as the whole process is taking three days (!) I have multiple other apps open that I'm working on. I'd like to be able to position the welcome window at the edge of the screen or in a corner, but every time I open a new bolus of dpxes, the welcome window reverts to the upper center of the screen. Thus I have to keep the center of my screen open, or the welcome window gets lost beneath other windows. 
![Screen Shot 2020-03-18 at 5 02 10 PM](https://user-images.githubusercontent.com/2841957/77018616-5b104000-693b-11ea-8bb8-631ea9ae535a.jpg)

**Comment by gregcotten** [comment]
Might need to make this a preference! In the meantime, I suggest dragging to the Screen icon in the dock - it will perform the same task!

**Comment by adamwilt** [comment]
Sure. But the dock icon is a smaller drop target, and the welcome window shows me the ten previous dpx sequences, so it's a handy visual check that my sequencing is correct and I haven't missed a batch. I know, I know: first-world problems...  ;-)
